src/scripts/pokemons/PokemonList.ts

```diff
@@ -21322,29 +21322,6 @@ const pokemonList = createPokemonArray(
 const pokemonNameIndex = {};
 const maxEggCycles = Math.max(...pokemonList.map(p => p.eggCycles));
 
-// TODO move to its own initialize method that gets called on game start.
-pokemonList.forEach(p => {
-    const baseOffense = 2 * Math.round(Math.sqrt(p.base.attack * p.base.specialAttack) + Math.sqrt(p.base.speed));
-    const baseDefense = 2 * Math.round(Math.sqrt(p.base.defense * p.base.specialDefense) + Math.sqrt(p.base.speed));
-    const baseStamina = 2 * p.base.hitpoints;
-
-    (p as PokemonListData).attack = Math.max(10, Math.floor(Math.sqrt(baseDefense * baseStamina) * baseOffense / 250));
-    if ((p as PokemonListData).baby) {
-        (p as PokemonListData).evolutions?.forEach(evo => {
-            pokemonBabyPrevolutionMap[evo.getEvolvedPokemon()] = evo.basePokemon;
-            const poke = pokemonList.find(_p => _p.name == evo.getEvolvedPokemon());
-            p.eggCycles = Math.round(poke.eggCycles * 0.8);
-        });
-    } else {
-        (p as PokemonListData).evolutions?.forEach(evo => {
-            const poke = pokemonList.find(_p => _p.name == evo.getEvolvedPokemon());
-            poke.eggCycles = Math.min(maxEggCycles, Math.round(p.eggCycles * 1.5));
-        });
-    }
-    (p as PokemonListData).nativeRegion = (p as PokemonListData).nativeRegion || GameConstants.MaxIDPerRegion.findIndex(maxRegionID => maxRegionID >= Math.floor(p.id));
-    pokemonNameIndex[p.name.toLowerCase()] = p;
-});
-
```

Comment:
I'm not sure if this will affect `pokemonMap` or not, I'll try give this a test in the morning though.

Otherwise as far as I can tell everything looks good 👍